index.html

```diff
@@ -14,6 +14,13 @@
         <a href="/">
             <h1 class="nav-brand">My Logo</h1>
         </a>
+        <nav class="header-nav">
+            <ul class="nav-list">
+                <li class="nav-item"><a href="#" >Portoflio</a></li>
+                <li class="nav-item"><a href="#" >About</a></li>
+                <li class="nav-item"><a href="#" >Contact</a></li>
+            </ul>
+        </nav>
         <img src="images/icons/Union.svg" alt="Menu" />
     </header>
     <section class="headerline">
```

Comment:
- [ ] Please, consider fixing the spaces in the headline and contact section also 👍 
<img width="525" alt="Screen Shot 2022-02-18 at 00 37 36" src="https://user-images.githubusercontent.com/49064106/154567219-4f9b0ef0-1175-4e84-85c2-39941b332752.png">

<img width="513" alt="Screen Shot 2022-02-18 at 00 37 59" src="https://user-images.githubusercontent.com/49064106/154567241-b3f6dc28-cfd2-4c45-9acb-6b8916e64970.png">